schemas/responses/CustomFieldResponse.yaml

```diff
@@ -0,0 +1,6 @@
+allOf:
```

Comment:
I suggest avoiding duplication in a path: `/resposes/` and `*Response.yaml`

just simplify it to the `schemas/responses/CustomField.yaml` or `schemas/responses/CustomField.success.yaml`